.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

```suggestion
logs/*
```
this will match all logs in the folder